packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/classes_table_diff.dart

```diff
@@ -199,23 +199,17 @@ class ClassesTableDiff extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ValueListenableBuilder(
-      valueListenable: classes.filter,
-      builder: (_, __, ___) {
-        final data = classes.filtered();
-        final dataKey = 'ClassesTableSingle-$data';
-
-        return FlatTable<DiffClassStats>(
-          columns: _columns,
-          columnGroups: _columnGroups,
-          data: data,
-          dataKey: dataKey,
-          keyFactory: (e) => Key(dataKey),
-          selectionNotifier: selection,
-          defaultSortColumn: _retainedSizeDeltaColumn,
-          defaultSortDirection: SortDirection.descending,
-        );
-      },
+    final dataKey = 'ClassesTableSingle-${identityHashCode(classes)}';
```

Comment:
do we want to be uniquely identifying this table by the data set? I would think we'd want to preserve the sorting and sort directions for ClassesTableDiff no matter what the data passed to it is.

Reply:
Added TODO to address it later.